spring-cloud-starter-dataflow-collector-metrics/pom.xml

```diff
@@ -2,13 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<artifactId>spring-cloud-starter-stream-collector-metrics</artifactId>
+	<artifactId>spring-cloud-starter-dataflow-collector-metrics</artifactId>
```

Comment:
Do we still have to have "collector-metrics" vs.  "metrics-collector"  the former is in the name of the repo, so it kinda looks odd to have the artifact with reversed names.   If we can't, then we could rename/create a new repo.